src/main/java/com/google/sps/servlets/LatLngServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.inject.Inject;
+
+import com.google.sps.data.GeoCoding;
+import com.google.gson.Gson;
+
+@WebServlet("/latlng")
+public class LatLngServlet extends HttpServlet {
+  GeoCoding GeoCodingObject;
+
+  @Inject
+  LatLngServlet(GeoCoding GeoCodingObject) {
+    this.GeoCodingObject = GeoCodingObject;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String location = request.getParameter("location");
+    response.setContentType("application/json");
+    response.getWriter().println(new Gson().toJson(GeoCodingObject.fromAddressToLatLng(location)));
```

Comment:
I think this fails when returned null from GeoCodingObject

Reply:
Maybe it would worth adding a check here not just inside of fromAddressToLatLng. Also what if there is no address because the location is the whole city?

Reply:
It will return coordinates of some point in this city